Big-Spender/readme.md

```diff
@@ -102,9 +115,11 @@ INSERT YOUR QUERY HERE
 ```sql
 INSERT YOUR QUERY HERE
 ```
+select sum(spends.amount), suppliers.supplier from spends inner join suppliers on spends.supplier_id = suppliers.id group by suppliers.supplier;
```

Comment:
nice join for the suppliers, again we can look at the alias potentially, but overall the query is good. Although no move caps on keywords?